webpack/stories/docs/ClientRouting.mdx

```diff
@@ -0,0 +1,97 @@
+import { Meta } from '@theforeman/stories';
+
+<Meta
+  title="Introduction|Client Routing"
+  parameters={{
+    storyWeight: 120,
```

Comment:
A story with a `storyWeight` of 120 already exists: https://github.com/theforeman/foreman/blob/e7c0ad82a2f327e105f3ceae62b2b6721783ae5a/webpack/stories/docs/creating-a-form.stories.mdx

Please pick a unique story-weight to represent the story order in the stories list.